bindings/matrix-sdk-ffi/src/timeline/content.rs

```diff
@@ -354,16 +354,28 @@ impl From<&matrix_sdk_ui::timeline::AnyOtherFullStateEventContent> for OtherStat
             }
             Content::RoomPinnedEvents(_) => Self::RoomPinnedEvents,
             Content::RoomPowerLevels(c) => {
-                let changes = match c {
+                let previous: Option<HashMap<String, i64>>;
+                let users: HashMap<String, i64>;
+                match c {
                     FullContent::Original { content, prev_content } => {
-                        power_level_user_changes(content, prev_content)
+                        previous = prev_content.as_ref().map(|prev_content| {
```

Comment:
style nit: can you have each branch return a `Self::RoomPowerLevels` immediately, instead of having two temporary variables? it would read slightly more straightforward.

Reply:
Sure, I just did it this way to not repeat too much code.

Reply:
Done in https://github.com/matrix-org/matrix-rust-sdk/pull/3199/commits/ff911e4125684f6a12aa008911af99b6e1b6e21c